src/components/FromSTIX/StatisticsObjects.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import styles from "./from_stix.module.scss";
+
+const StatisticsObjects = (props) => {
+  const statistic = (props.objectsCount / props.fieldsCount) * 100;
+
+  return (
+    <div className="bx--col">
+      <div className="bx--label-description">{props.type} STIX Objects</div>
+      <div className={styles.coverage_precent}>
+        {props.objectsCount == 0 ? 0 : Math.round(statistic * 100) / 100} %
```

Comment:
js equality preferable use is strict equality, "===",
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness
please check ESLint warnings before committing
![image](https://user-images.githubusercontent.com/16198896/127462596-d54c87bd-6b80-4b1f-a6d6-ede87c108e82.png)